package.json

```diff
@@ -54,11 +56,13 @@
 		"@types/node": "^14.14.33",
 		"@types/resolve": "^1.20.1",
 		"lint-staged": "^10.5.4",
+		"magic-string": "^0.30.5",
 		"preact": "^10.19.2",
 		"preact-iso": "^2.3.2",
 		"preact-render-to-string": "^6.3.1",
 		"prettier": "^2.2.1",
 		"rimraf": "^3.0.2",
+		"rollup": "^2.77.3",
```

Comment:
Done just for types -- newer versions of Vite (conveniently) expose Rollup's types (`import type { Rollup } from 'vite'`) so we can drop this in the future.